corehq/apps/userreports/dbaccessors.py

```diff
@@ -27,21 +27,28 @@ def get_report_configs_for_domain(domain):
     )
 
 
-def get_datasources_for_domain(domain, referenced_doc_type=None):
-    from corehq.apps.userreports.models import DataSourceConfiguration
+def get_datasources_for_domain(domain, referenced_doc_type=None, include_static=False):
+    from corehq.apps.userreports.models import DataSourceConfiguration, StaticDataSourceConfiguration
     key = [domain]
     if referenced_doc_type:
         key.append(referenced_doc_type)
-    return sorted(
+    datasources = sorted(
         DataSourceConfiguration.view(
             'userreports/data_sources_by_build_info',
             start_key=key,
             end_key=key + [{}],
             reduce=False,
             include_docs=True
         ),
-        key=lambda config: config.display_name
-    )
+        key=lambda config: config.display_name)
+
+    if include_static:
+        static_ds = StaticDataSourceConfiguration.by_domain(domain)
+        if referenced_doc_type:
+            static_ds = [ds for ds in static_ds if ds.referenced_doc_type == referenced_doc_type]
+        datasources.extend(sorted(static_ds, key=lambda config: config.display_name))
+
+    return sorted(datasources, key=lambda config: config.display_name)
```

Comment:
No need to sort on line 35 or line 49, because you're sorting here.

Reply:
Thanks for catching. I meant to remove this line. It's not necessary now, but in most places we have normal data sources ordered before static ones. If we move to using this more generally, then it will keep the same behavior by sorting twice